### ISSUE TITLE
feat: write API to send data from the routing layer and receive it at an IOx server

### ISSUE THREAD

**Issue by pauldix** [influxdata/influxdb_iox#1088]
The routing layer will need to write entries to remote IOx servers. The remote IOx server should assign a clock value and writer ID to the entry (turning it into a `SequencedEntry`). This issue is to implement an API on the receiving side and updating the client to send.

If this server is acting as a Write Buffer it should buffer and potentially replicate the write to its peers (this part is for another issue). Or if it is running a Mutable Buffer it should write to that (also for another issue).

**Comment by pauldix** [comment]
#1157 will need to be done before this one.

**Comment by carols10cents** [comment]
So I'm pretty convinced that this is a duplicate of #920 and that this is done. @mkmik or @pauldix, if I'm incorrect, please let me know what part of this *isn't* done yet :)
